specification/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,39 @@
+# Pricing Model
+
+Pricing Model indicates whether a charge is a fixed, on-demand or discounted.
+
+The PricingModel column MUST be present. This column MUST be of type String and MUST be one of the Supported Values.
+
+## Column ID
+
+PricingModel
+
+## Display name
+
+Pricing Model
+
+## Description
+
+TBD once definition is confirmed
+
+## Content constraints
+
+|    Constraint   |      Value       |
+|:----------------|:-----------------|
+| Column required | True             |
+| Data type       | String           |
+| Allows nulls    | False             |
+| Value format    | List of values   |
+
+Allowed values:
```

Comment:
Should we add a "Shared" option to account for dedicated host for AWS?

Reply:
IMO we should not add it now and we can always add it in a point release once we have feedback from it's use in the wild. It's easier to add then to remove/update. Will close this for now